### PR TITLE
docs(module-tools): add faq about how to skip less or scss processing

### PR DESCRIPTION
**Pull request by 10Derozan**
## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/module-doc/docs/en/guide/faq/build.mdx [hunk 1]
```diff
@@ -212,7 +212,7 @@ For source code files, it is recommended to fix the type problem, which can bett
 
 1. enable [skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck) to skip the d.ts check of the thrid-party package.
 2. If the package exports ts files directly, skipLibCheck will not work because it can only skip the d.ts check,
-so you can only turn off [dts.abortOnError](/api/config/build-config.html#dtsabortonerror) to ignore these errors.
+   so you can only turn off [dts.abortOnError](/api/config/build-config.html#dtsabortonerror) to ignore these errors.
```
Comment (chenjiahan) [inline packages/document/module-doc/docs/en/guide/faq/build.mdx:215]: should we add an empty line before this section

File: packages/document/module-doc/docs/zh/guide/faq/build.mdx [hunk 2]
```diff
@@ -235,6 +235,36 @@ import BuildOtherFAQ from '@site-docs/components/faq-build-other';
 
 <BuildOtherFAQ />
 
+### bundleless 如何跳过对 less / scss 文件的预处理
+
+bundleless 是单文件编译的方式，只需要将你的 less / scss 文件从入口里移除并且将其拷贝到产物里即可。
+注意我们还会把 js 引用 less / scss 的 moduleId 进行改写，通过 [redirect](/api/config/build-config#redirect) 插件关闭它。
```
Comment (chenjiahan) [inline packages/document/module-doc/docs/zh/guide/faq/build.mdx:241]:
```suggestion
注意我们还会把 js 引用 less / scss 的 moduleId 进行改写，你可以通过 [redirect](/api/config/build-config#redirect) 配置关闭它。
```

File: packages/document/module-doc/docs/en/guide/faq/build.mdx (hunk 1 above)
Comment (10Derozan) [reply packages/document/module-doc/docs/en/guide/faq/build.mdx:215]: why

Comment (chenjiahan) [reply packages/document/module-doc/docs/en/guide/faq/build.mdx:215]: Or maybe should be in the same line?
image

Comment (10Derozan) [reply packages/document/module-doc/docs/en/guide/faq/build.mdx:215]: yes,they are in the same line before.

**Pull request merged**